tests/cli/output/universalindent.cfg

```diff
@@ -15,11 +15,11 @@ parameterOrder=ipo
 showHelpParameter=-h
 stringparaminquotes=false
 useCfgFileParameter="-c "
-version=Uncrustify_d-0.67-439-48bcf172
+version=Uncrustify-0.67-454-af63f642
```

Comment:
This needs to be stripped (like we do for the other tests), or it is going to change every single commit.

Reply:
Yes, but some more also...